### ISSUE TITLE
Playready is not supported in MS Edge (specific versions) from v2.5.7 

### ISSUE THREAD

**Issue by glhvta** [google/shaka-player#2413]
**Have you read the [FAQ](https://bit.ly/ShakaFAQ) and checked for duplicate open issues?**
Yes

**What version of Shaka Player are you using?**
v2.5.9

**Can you reproduce the issue with our latest release version?**
yes

**Can you reproduce the issue with the latest code from `master`?**
yes

**Are you using the demo app or your own custom app?**
both

**If custom app, can you reproduce the issue using our demo app?**
yes

**What browser and OS are you using?**
Windows 10 Enterprise
Version 1803
OS Build 17134.1184

Microsoft Edge 42.17134.1098.0
Microsoft EdgeHTML 17.17134

**What are the manifest and license server URIs?**

The issue is reproduced on all the tests assets with Playready drm
https://shaka-player-demo.appspot.com/demo/#audiolang=en-US;textlang=en-US;uilang=en-US;asset=https://demo.unified-streaming.com/video/tears-of-steel/tears-of-steel-dash-playready.ism/.mpd;panel=SEARCH;build=uncompiled

**What did you do?**

call `shaka.Player.probeSupport()

**What did you expect to happen?**
Playready drm is supported

**What actually happened?**
Playready drm is not supported
![image](https://user-images.githubusercontent.com/31937370/75055680-8ba0bd80-54e6-11ea-94a0-47205abfc25f.png)

**Issue description**

Recently we faced an issue with PlayReady DRM playback on a specific version of the legacy MS Edge browser (42.17134.1098.0) that playready drm was is not supported. 

This issue was introduced in v2.5.7 version of shaka player after integration of eme-encryption-scheme-polyfill.

After debugging it was observed that the issue occurs in eme-encryption-scheme-polyfill in hasEncryptionScheme check. When there is no video or audio capabilities it will throw an error
```
Unable to get property 'encryptionScheme' of undefined or null reference
```

On the specified version of Edge Microsoft Edge 42.17134.1098.0 this issue is reproduced.
please see 
![image](https://user-images.githubusercontent.com/31937370/75056115-619bcb00-54e7-11ea-891e-7f53afd0a4d7.png)

Seems like it is a known issue that Edge does not report correct capabilities and shaka-player has already implemented a fix for this case
https://github.com/google/shaka-player/blob/b38b75c892a157245890c35b62d80beade484f4e/lib/media/drm_engine.js#L546-L555

There is an issue in eme-encryption-scheme-polyfill repo 
https://github.com/google/eme-encryption-scheme-polyfill/issues/15

This issue is not reproduced in Microsoft Edge 44.18362.449.0
So probably this is an issue of MS Edge version 42.17134.1098.0.
We know that shaka player supports only the latest version, but I guess it would be nice to upgrade the eme-encryption-scheme-polyfill module after the issue would be resolved.

Thanks!

**Comment by joeyparrish** [comment]
This will be fixed in v2.0.1 of the polyfill once https://github.com/google/eme-encryption-scheme-polyfill/pull/14 is merged.  Then we can update Shaka Player to depend on v2.0.1 of the polyfill and cherry-pick that change to Shaka Player v2.5.10.

Thanks for the report and the PR!

**Comment by glhvta** [comment]
Good news!
Thank you!
